sdk/storage/Azure.Storage.Files.DataLake/src/Models/PathAccessControlEntry.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+
+namespace Azure.Storage.Files.DataLake.Models
+{
+    /// <summary>
+    /// Represents an access control in a file access control list.
+    /// </summary>
+    public class PathAccessControlEntry
```

Comment:
Nit - With how the API looks now, should this just be `PathAccessControl`?

Reply:
No, because PathAccessControl is already a class containing Owner, Group, PathPermissions, and an IEnumerable<PathAccessControlEntry>.  It is returned by PathClient.GetAccessControl().

Reply:
`Entry` as a suffix is just a little odd.  Can we change it to `PathAccessControlItem` since all of our other `Foo` methods return `FooItem`s?

Reply:
fixed.